interview-questions.md

```diff
@@ -9,48 +9,57 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is the difference between a parameter and an argument?
 
   Your answer:
-
+  An argument belongs to a method and a parameter is just a placeholder within a function. A method's argument is in the parentheses like (.includes()) and it tells the method the specifics for that method. Parameters are the placeholders within a function that you put your given values into.
```

Comment:
To expand a little on arguments, arguments are not confined to only methods.  They are passed in when a function is invoked which then get renamed as the parameter when inside the function.